pupil_src/shared_modules/calibration_routines/calibrate.py

```diff
@@ -41,9 +41,10 @@ def calibrate_2d_polynomial(cal_pt_cloud,screen_size=(1,1),threshold = 35, binoc
         logger.info('first iteration. root-mean-square residuals: {}, in pixel'.format(err_rms))
         logger.info('second iteration: ignoring outliers. root-mean-square residuals: {} in pixel'.format(new_err_rms))
 
-        logger.info('used {} data points out of the full dataset {}: subset is {} percent'.format(
-            (cal_pt_cloud[err_dist <= threshold].shape[0], cal_pt_cloud.shape[0],
-             100*float(cal_pt_cloud[err_dist <= threshold].shape[0])/cal_pt_cloud.shape[0])))
+        used_no = cal_pt_cloud[err_dist <= threshold].shape[0]
+        complete_no = cal_pt_cloud.shape[0]
```

Comment:
Variable name. Consider `used_num` or `used_number`. `used_no` may be misleading to some readers as `no` can be read as a negation in English at least.